source/common/http/http1/conn_pool.cc

```diff
@@ -76,7 +76,7 @@ void ActiveClient::StreamWrapper::onDecodeComplete() {
     parent_.codec_client_->close();
   } else {
     auto* pool = &parent_.parent();
-    pool->dispatcher().post([pool]() -> void { pool->onUpstreamReady(); });
```

Comment:
There is similar code in source/common/tcp/conn_pool.cc, see ActiveTcpClient::clearCallbacks()

CI seems to have caught it:
2020-12-01T14:22:57.8915160Z source/common/tcp/conn_pool.cc:52:54: error: 'onUpstreamReady' is a protected member of 'Envoy::ConnectionPool::ConnPoolImplBase'
2020-12-01T14:22:57.8916126Z     pool->dispatcher().post([pool]() -> void { pool->onUpstreamReady(); });
2020-12-01T14:22:57.8916661Z                                                      ^
2020-12-01T14:22:57.8917589Z bazel-out/k8-opt/bin/source/common/conn_pool/_virtual_includes/conn_pool_base_lib/common/conn_pool/conn_pool_base.h:245:8: note: declared protected here
2020-12-01T14:22:57.8918237Z   void onUpstreamReady();

Reply:
thanks, moving it over. it breaks tcp tests for reasons beyond me. working on why.